docs/unleash-context.md

```diff
@@ -0,0 +1,22 @@
+# Unleash Context
+
+In order to standardize a few activation strategies we also needed to 
```

Comment:
I'm probably should stick to British, but I love to use "z" instead of "s" in wordz, just a bad habit.